### PR TITLE
SEED 731: Profiles template update 

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@miltontony @saeedmb8 please review

Smart Design 

screen shot 2016-07-14 at 1 26 53 pm

Feature Design 

screen shot 2016-07-14 at 1 26 20 pm

File: iogt/templates/registration/password_reset_complete.html [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 {% load i18n %}
-
-
 {% block body %}
- <p>{% trans "Your password has been set.  You may go ahead and log in now." %}</p>
-
- <p><a href="{% url 'login' %}">{% trans 'Log in' %}</a></p>
-
+<div class="profiles">
+	<p>{% trans "Your password has been set.  You may go ahead and log in now." %}</p>
```
Comment (smarzban) [inline iogt/templates/registration/password_reset_complete.html]: @Mitso there is two white spaces after `set.`

File: iogt/templates/registration/password_reset_confirm.html [hunk 2]
```diff
@@ -1,23 +1,25 @@
...
+			</fieldset>
+			<fieldset>
+				<label for="id_new_password2">{% trans 'Confirm password:' %}</label>
+				{{ form.new_password2.errors }}
+				{{ form.new_password2 }}
+			</fieldset>
+			<input type="submit" value="{% trans 'Change my password' %}" />
+		</form>
+	{% else %}
+	<p>{% trans "The password reset link was invalid, possibly because it has already been used.  Please request a new password reset." %}</p>
```
Comment (smarzban) [inline iogt/templates/registration/password_reset_confirm.html]: @Mitso after `used.` as well

File: iogt/static/css/modules/_m-search.scss [hunk 3]
```diff
@@ -48,6 +54,4 @@
 		font-size: $font-xx-large;
 }
 
-.search--result__subtitle {
-
-}
\ No newline at end of file
+.search--result__subtitle {}
```
Comment (smarzban) [inline iogt/static/css/modules/_m-search.scss]: @Mitso can you please add a new line at the end of the file

**Comment by smarzban** [comment]
@Mitso left some small comments otherwise 👍  🍰

**Comment by Mitso** [comment]
Awesome, thanks @saeedmb8

**Pull request merged**